### PR TITLE
[15.0][MIG] stock_picking_line_sequence

### PR DESCRIPTION
**Pull request by DavidJForgeFlow**
Standard Migration to 15.0

Supersedes #979  

@ForgeFlow @slominchar

### CODE REVIEW

File: stock_picking_line_sequence/readme/CONTRIBUTORS.rst [hunk 1]
```diff
@@ -1,4 +1,5 @@
 
+* Sulivan Lominchar <slominchar@archeti.com>
```
Comment (LoisRForgeFlow) [inline stock_picking_line_sequence/readme/CONTRIBUTORS.rst]: Can you move sulivan to the end of the list?

Comment (DavidJForgeFlow) [reply stock_picking_line_sequence/readme/CONTRIBUTORS.rst]: done

**Review by LoisRForgeFlow — APPROVED** [review]

File: stock_picking_line_sequence/models/stock.py [hunk 2]
```diff
@@ -35,22 +35,25 @@ class StockMoveLine(models.Model):
...
-                description = False
-            uom = move_line.product_uom_id
-            line_key = (
-                str(move_line.product_id.id)
-                + "_"
-                + name
-                + (description or "")
-                + "uom "
-                + str(uom.id)
+            line_key, name, description, uom = get_aggregated_properties(
```
Comment (rousseldenis) [inline stock_picking_line_sequence/models/stock.py]: Am I wrong or some variables are not used ?

Comment (DavidJForgeFlow) [reply stock_picking_line_sequence/models/stock.py]: Totally true, I just remove the unused variables. Thanks!

**Comment by LoisRForgeFlow** [comment]
@rousseldenis good to go?

**Review by AaronHForgeFlow — APPROVED** [review]
LGTM!

**Comment by rousseldenis** [comment]
@LoisRForgeFlow We need to wait for https://github.com/OCA/odoo-test-helper/pull/24

**Pull request merged**